aztk/spark/models/plugins/install/install.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+apt-get update
+
+for PACKAGE in "$@"
+do
+    eval $COMMAND $PACKAGE
```

Comment:
also for apt I think as @jafreck noticed it might be more performant to run apt install -y package1 package2, etc instead of multiple apt instance
(I think you should be able to do the same for pip and conda)

Reply:
Will only do one instance